<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedTab</class>
 <widget class="QWidget" name="AdvancedTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="remote_group">
     <property name="title">
      <string>Remote Type</string>
     </property>
     <layout class="QVBoxLayout" name="remote_layout">
      <item>
       <widget class="QRadioButton" name="remote_https_radio">
        <property name="text">
         <string>HTTPS</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="remote_ssh_radio">
        <property name="text">
         <string>SSH</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="options_group">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="options_layout">
      <item>
       <widget class="QCheckBox" name="open_browser_check">
        <property name="text">
         <string>Open repository in browser after creation</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
